client/config-service-rest-client/src/main/java/org/hawkular/apm/config/service/rest/client/ConfigurationServiceRESTClient.java

```diff
@@ -70,115 +62,17 @@ public ConfigurationServiceRESTClient() {
      */
     @Override
     public CollectorConfiguration getCollector(String tenantId, String type, String host, String server) {
-        if (log.isLoggable(Level.FINEST)) {
-            log.finest("Get collector configuration: tenantId=[" + tenantId + "] type=[" + type + "] host=[" + host
-                    + "] server=[" + server + "]");
-        }
-
-        // Check if configuration provided locally
-        if (PropertyUtil.getProperty(ConfigurationLoader.HAWKULAR_APM_CONFIG) != null) {
-            CollectorConfiguration ret = ConfigurationLoader.getConfiguration(type);
```

Comment:
I think some of the tests may use local configuration, so need this to be checked before doing a remote call.